From e8780c02d307380e9e08d1b85888a599d53078ac Mon Sep 17 00:00:00 2001
From: Andy Green <andy@openmoko.com>
Date: Thu, 7 Aug 2008 16:27:21 +0100
Subject: [PATCH] config-remove-evbug-module.patch

Carsten mentioned that evbug module is getting modprobed in
to ASU somehow and making havoc.  This patch stops us building
it because there's no time to figure out why it gets modprobed
in and it doesn't seem to do anything but spew syslog -- and we
never needed it before now.

Reported-by: Carsten Haitzler <raster@openmoko.org>
Signed-off-by: Andy Green <andy@openmoko.com>
---
 defconfig-2.6.24            |    2 +-
 defconfig-2.6.24-maxmodules |    2 +-
 defconfig-gta01             |    2 +-
 defconfig-gta02             |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/defconfig-2.6.24 b/defconfig-2.6.24
index 0bcf50a..52d5e45 100644
--- a/defconfig-2.6.24
+++ b/defconfig-2.6.24
@@ -906,7 +906,7 @@ CONFIG_INPUT_MOUSEDEV_SCREEN_X=480
 CONFIG_INPUT_MOUSEDEV_SCREEN_Y=640
 # CONFIG_INPUT_JOYDEV is not set
 CONFIG_INPUT_EVDEV=y
-CONFIG_INPUT_EVBUG=m
+# CONFIG_INPUT_EVBUG is not set
 
 #
 # Input Device Drivers
diff --git a/defconfig-2.6.24-maxmodules b/defconfig-2.6.24-maxmodules
index ec94668..d4c1988 100644
--- a/defconfig-2.6.24-maxmodules
+++ b/defconfig-2.6.24-maxmodules
@@ -867,7 +867,7 @@ CONFIG_INPUT_MOUSEDEV_SCREEN_X=480
 CONFIG_INPUT_MOUSEDEV_SCREEN_Y=640
 # CONFIG_INPUT_JOYDEV is not set
 CONFIG_INPUT_EVDEV=y
-CONFIG_INPUT_EVBUG=m
+# CONFIG_INPUT_EVBUG is not set
 
 #
 # Input Device Drivers
diff --git a/defconfig-gta01 b/defconfig-gta01
index 3320560..6f36712 100644
--- a/defconfig-gta01
+++ b/defconfig-gta01
@@ -867,7 +867,7 @@ CONFIG_INPUT_MOUSEDEV_SCREEN_X=480
 CONFIG_INPUT_MOUSEDEV_SCREEN_Y=640
 # CONFIG_INPUT_JOYDEV is not set
 CONFIG_INPUT_EVDEV=y
-CONFIG_INPUT_EVBUG=m
+# CONFIG_INPUT_EVBUG is not set
 
 #
 # Input Device Drivers
diff --git a/defconfig-gta02 b/defconfig-gta02
index 376d53e..0838dad 100644
--- a/defconfig-gta02
+++ b/defconfig-gta02
@@ -866,7 +866,7 @@ CONFIG_INPUT_MOUSEDEV_SCREEN_X=480
 CONFIG_INPUT_MOUSEDEV_SCREEN_Y=640
 # CONFIG_INPUT_JOYDEV is not set
 CONFIG_INPUT_EVDEV=y
-CONFIG_INPUT_EVBUG=m
+# CONFIG_INPUT_EVBUG is not set
 
 #
 # Input Device Drivers
-- 
1.5.6.5

